qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -264,6 +264,8 @@ def delete_mds_coredump(self, daemon_id):
         if core_dir:  # Non-default core_pattern with a directory in it
             # We have seen a core_pattern that looks like it's from teuthology's coredump
             # task, so proceed to clear out the core file
+            if core_dir[0] == '|':
+                core_dir = core_dir[1:]
```

Comment:
This is not right. `|...` indicates a program the kernel sends the core dump too. We should skip cleaning up the coredump in that case I suppose.

Reply:
Yeah, you are right.  Let's just skip it in this case.